docs/src/userguide/special_methods.rst

```diff
@@ -122,7 +122,12 @@ of the superclass will always be called, even if it is overridden.  This is in
 contrast to typical Python behavior where superclass methods will not be
 executed unless they are explicitly called by the subclass.
 
-.. Note:: There is no :meth:`__del__` method for extension types.
+Python 3.4 introduced the :meth:`__del__` method to allow for the safe
```

Comment:
Maybe "Python 3.4 made it possible for extension types to safely define finalizers for objects. When running a Cython module on Python 3.4 and higher you can add..."
then at the end "On Python <3.4 :meth:`__del__` will not be called".

Obviously `__del__` has existing as a Python concept for much longer and so my proposed wording tries to avoid that confusion.

Reply:
OK, I did not know that. Let me just fix that...